ini-templates/base-template.ini

```diff
@@ -15,6 +15,8 @@ postgresql.statement_timeout = 120
 pyramid.default_locale_name = en
 sqlalchemy.url = ${pg_uri}
 retry.attempts = 3
+remote_indexing = ${remote_indexing}
+remote_indexing_threshold = 50
```

Comment:
what does this do?

Reply:
It's used in 
https://github.com/ENCODE-DCC/snovault/blob/SNO-159-allow-remote-indexing/src/snovault/elasticsearch/indexer.py#L157

to determine if indexing should be remote or local based on the number of uuids to index.  There are 8 local indexing processes available, down from 16.

50 was for troubleshooting.  This has been updated to 10000 to start.  We're going to test a full reindexing on 8 procs over the weekend.  This could probably be higher but it really depends on the object type.  10000 files or experiments should probably be offloaded.